packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -269,6 +270,16 @@ class IconButton extends StatelessWidget {
   /// used for accessibility.
   final String tooltip;
 
+  /// Whether detected gestures should provide acoustic and/or haptic feedback.
+  ///
+  /// For example, on Android a tap will produce a clicking sound and a
+  /// long-press will produce a short vibration, when feedback is enabled.
```

Comment:
It seems that `RawMaterialButton`, `MaterialButton` and `IconButton` do not have an `onLongPress` callback or option, which makes this inaccurate. We should either modify the docs to explain that long-press is not supported for these widgets, or implement them correctly. 

@HansMuller What seems like the appropriate step here?

Reply:
Related to this issue somewhat, since it explains the default long press behavior for Material buttons and the intent to add an `onLongPress` param: https://github.com/flutter/flutter/pull/40641

Reply:
Removed comment in doc about long-press

Reply:
I don't think just removing it is the right approach -- I think if we take the route of not having it respond to long presses, we need to explain how this is different from what is expected. Let's wait for @HansMuller before we continue forward about this topic

Reply:
I think that #40641 will land pretty soon and so it's OK to explain that enableFeedback does what InkWell.enableFeedback does without qualification.

Reply:
Done